django/contrib/admin/options.py

```diff
@@ -850,9 +850,12 @@ def action_checkbox(self, obj):
     def _get_base_actions(self):
         """Return the list of actions, prior to any request-based filtering."""
         actions = []
+        local_action_names = [self.get_action(action)[1] for action in self.actions or []]
```

Comment:
Use a `set` if this value is only used for containment checks

```suggestion
        local_action_names = {self.get_action(action)[1] for action in self.actions} if self.actions else set()
```

Reply:
I don't have a strong feeling about either, this approach avoids an unnecessary intermediary list creation though.